features/zoek-met-geslachtsnaam-geboortedatum.feature

```diff
@@ -2,522 +2,416 @@
+    | Geslachtsnaam is meer dan 200 karakters lang | maxLength | Waarde is langer dan maximale lengte 200.                                                           | abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ |
+    | Geslachtsnaam bevat ongeldige karakters      | pattern   | Waarde voldoet niet aan patroon ^[a-zA-Z0-9À-ž \.\-\']{1,200}$\|^[a-zA-Z0-9À-ž \.\-\']{3,199}\*{1}$ | <script>alert('hello world');</script>                                                                                                                                                                              |
+
+  @fout-case
+  Scenario: Meerdere invalide parameters
+    Als personen wordt gezocht met de volgende parameters
+    | naam          | waarde                                 |
+    | type          | ZoekMetGeslachtsnaamEnGeboortedatum    |
+    | geslachtsnaam | <script>alert('hello world');</script> |
+    | geboortedatum | 19830526                               |
+    | fields        | burgerservicenummer                    |
+    Dan bevat de response de volgende gegevens
+    | naam     | waarde                                                                                                      |
+    | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+    | title    | Een of meerdere parameters zijn niet correct.                                                               |
+    | status   | 400                                                                                                         |
+    | detail   | De foutieve parameter(s) zijn: geboortedatum, geslachtsnaam.                                                |
+    | code     | paramsValidation                                                                                            |
+    | instance | /personen                                                                                                   |
+    En bevat de response een invalidParams met de volgende gegevens
+    | code    | name          | reason                                                                                              |
+    | pattern | geslachtsnaam | Waarde voldoet niet aan patroon ^[a-zA-Z0-9À-ž \.\-\']{1,200}$\|^[a-zA-Z0-9À-ž \.\-\']{3,199}\*{1}$ |
+    | date    | geboortedatum | Waarde is geen geldige datum.                                                                       |
 
 Rule: Bij zoeken met de "*" wildcard moet minimaal 2 letters (exclusief de wildcard teken) worden opgegeven
       De "*" wildcard komt overeen met nul of meer (niet-spatie) karakters
 
```

Comment:
dat is minimaal 3 tekens geworden

Reply:
wat bedoel je met "niet-spatie"?

Als ik bijvoorbeeld zoek met "gaa*", vind ik dan "Gaay Fortman" niet wegens de spatie?

Reply:
ik heb dit overgenomen uit de wildcard.feature in Haal Centraal Common.